getvhostmap.lua

```diff
@@ -0,0 +1,48 @@
+-- A simple Lua script which serves up the HAProxy
+-- vhost to backend map file.
+function check_file_exists(name)
+   local f=io.open(name,"r")
+   if f~=nil then io.close(f) return true else return false end
+end
+
+function read_config_file(cmdline)
```

Comment:
Can you replace instances of `config` with `vhostmap`? It's a little confusing when reading the code, because it implies that's `haproxy.cfg`.